airflow/kubernetes/pod.py

```diff
@@ -50,13 +50,13 @@ class Pod:
     :param secrets: Secrets to be launched to the pod
     :type secrets: list[airflow.contrib.kubernetes.secret.Secret]
     :param result: The result that will be returned to the operator after
-        successful execution of the pod
+    successful execution of the pod
```

Comment:
I liked it more with indention, actually. Or was there in issue with that indention?

Reply:
We might need indentation for Sphinx. Or else the params won't be rendered properly.

Reply:
That was just me trying to figure out some sphinx errors. I can revert.